client-app/pages/account/order-details.vue

```diff
@@ -69,6 +69,41 @@
             {{ $t("pages.account.order_details.reorder_all_button") }}
           </VcButton>
 
+          <VcCard :title="$t('pages.account.order_details.loyalty_card.title')" :is-collapsible="true" class="mb-5">
```

Comment:
You can simple use `is-collapsible` here. It's not needed to explicitly pass "true" to boolean props.